services/php-fpm-nginx/Dockerfile

```diff
@@ -0,0 +1,36 @@
+ENV COMPOSER_NO_INTERACTION=1 \
+    COMPOSER_DISCARD_CHANGES=1 \
+    COMPOSER_ALLOW_SUPERUSER=1 \
+    DOCROOT=/usr/share/nginx/html
+
+# Configure php-fpm to run as a service.
+COPY files/service/php-fpm/run /root/php-fpm-run
+COPY files/tugboat.ini /root/tugboat.ini
+COPY files/nginx.conf /root/nginx.conf
+COPY files/sites-enabled/* /root/sites-enabled/
+
+RUN apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libmcrypt-dev libpng-dev libwebp-dev nginx && \
+    (apt-get install -y mysql-client || apt-get install -y mariadb-client) && \
+    docker-php-ext-configure gd --with-jpeg --with-freetype --with-webp && \
+    docker-php-ext-install -j$(nproc) gd iconv pdo_mysql && \
+    \
+    (((pecl install xdebug || pecl install xdebug-2.5.5 || pecl install xdebug-2.8.0beta2) && \
+    bash -c "(pecl install mcrypt <<< '' || pecl install mcrypt-1.0.1 <<< '' || docker-php-ext-install mcrypt || /bin/true)" && \
+    (docker-php-ext-enable mcrypt || /bin/true)) || /bin/true) && \
+    php -r "copy('https://raw.githubusercontent.com/composer/getcomposer.org/master/web/installer', 'composer-setup.php');" && \
+    php composer-setup.php && \
+    php -r "unlink('composer-setup.php');" && \
+    mv composer.phar /usr/local/bin/composer && \
+    mkdir -p /etc/service/php-fpm && \
+    mv /root/php-fpm-run /etc/service/php-fpm/run && \
+    mv /root/tugboat.ini /usr/local/etc/php/conf.d/tugboat.ini && \
+    mv ${DOCROOT} ${DOCROOT}.original && \
+    ln -s ${DOCROOT}.original ${DOCROOT} && \
+    mkdir -p /var/lib/tugboat/web && \
```

Comment:
This will cause conflicts when the repository is checked out to `/var/lib/tugboat`. What are you trying to achieve?

Reply:
This is probably a drupalism sneaking into this work. I think what this means is that we'd need to make the docroot dynamic somehow in the nginx config?